_posts/2022-03-23-blog-newsletter-social-media.md

```diff
@@ -0,0 +1,70 @@
+
+    For more information, please refer to the guideline on [Referencing]()
+
+- Lastly, you don’t have to wait until the end to communicate. While journal articles report on completed research output, social media communication can be used to communicate during the project, to solicit stakeholders interest, and to increase collaboration. 
+
+    *Be careful though, to not disclose scientific contributions before they are published as formal academic publications!*
+
+![BlogNewsletterSocialMedia_diagram](img/BlogNewsletterSocialMedia.jpg)
+
+## Specific pointers for different types of communication
+### Newsletter and blog posts
+-	In a ***newsletter***, summarize the research ‘news’ in a [Lay Summary][1]. Link and reference to the original work, and give attributions to contributors. 
+-	In a ***blog post***, you can write a longer popular science summary of the research, or describe some aspects of it – e.g. a case example, an impact it is already showing, or a story on where and how it was presented. Link and reference to the original work, and give attributions to contributors. 
+-	In *both newsletters and blogs*: ensure there is continuity. A blog with only one blog post is less efficient in communication than a blog where new blog posts are added every so often. The same goes for newsletters. It may be better to find or create a newsletter or blog with a broader scope, if news are regularly posted there. Include blog maintenance time in your plan to ensure that comments are answered and new blog posts/newsletters are published at the frequency you want. 
+
+### Social media. 
+Here, the purpose is to ping on the radar of a big audience, and make sure relevant stakeholders and interest groups notice your research. Therefore:
+
+- Go for short key messages, and insert a link to a blog post that describes the work in more detail (preferably) and/or to the original journal article/research output. 
+- ‘Mention’ contributors by naming them with links to their relevant social media profiles (important for both attribution and broader reach). 
+- Add relevant, subject-specific #hashtags, for example to the project or program that the work is part of, but also to topics that have established hashtags on the social media channel and are subject to living discussions or trending topics (obs: that are relevant). 
+
+Note that certain audiences, such as certain interest groups or people in a specific location/profession/etc. that are more important to target than others. The tweet that is seen by the largest (general) audience may not always be the one that is most impactful. Having your posts shared or re-tweeted by key members, or in important groups can be an effective way to reach a wide audience. Being active on the social media channels where you want to use, interacting with relevant people there, including re-sharing their content (that is of relevance to you and your work) can therefore be a helpful practice “in-between” your own postings.  
```

Comment:
```suggestion
Note that certain audiences, such as certain interest groups or people in a specific location/profession/etc. that are more important to target than others. The tweet that is seen by the largest (general) audience may not always be the one that is most impactful. Having your posts shared or re-tweeted by key members, or in important groups can be an effective way to reach a wide audience. Being active on the social media channels where you want to use, interacting with relevant people there, including re-sharing their content (that is of relevance to you and your work) can therefore be a helpful practice “in-between” your own postings.  
```
Same as above, not sure if this section is relevant for the guideline. I would suggest to remove it.
@willu47 what do you think?